API/api/tests/test_event.py

```
@@ -1,6 +1,117 @@
+        )
+
+        self.assertEqual(resp.status_code, 200)
+        self.event.refresh_from_db()
+        self.assertEqual(str(self.event.event_date), "2026-05-20")
+
+    def test_update_event_date_fails_when_booked_slot_exists(self):
+        self.login_and_set_cookies()
+
+        Slot.objects.create(
+            plage=self.plage,
+            start_at=datetime(2026, 5, 10, 9, 0),
```

Ruff (0.15.10)

[warning] `datetime.datetime()` called without a `tzinfo` argument

(DTZ001)